docs/relnotes.md

```diff
@@ -74,6 +74,13 @@ Published on 2024-03.
 [`taipy-templates` 3.1](https://pypi.org/project/taipy-templates/3.1.1/), and
 [`taipy-rest` 3.1](https://pypi.org/project/taipy-rest/3.1.1/) packages.
 
+!!! warning Error([#1180](https://github.com/Avaiga/taipy/issues/1180)) when downgrading from Taipy 3.1 to Taipy 3.0 
+    
+    If you are experiencing and error  dowgrading to Taipy 3.0, please 
+    reinstall Taipy with options `--no-cache-dir -I` like so: 
+      `pip install --no-cache-dir -I taipy==3.0.0`
+
+
```

Comment:
Since the issue appears when installing back `3.0`, this should be moved to line 200 in the `3.0` section.